src/index.html

```diff
@@ -11,7 +11,158 @@
+          >
+        </label>
+
+        <label class="form__field">
+          Name:
+          <input 
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form__field">
+          How old are You?
+          <input 
+            type="number" 
+            name="old"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form__field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="dateOfBirth"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+
+        <label class="form__field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__group">
+        <legend>Registration</legend>
+
+        <label class="form__field">
+          E-mail:
+          <input 
+            type="email" 
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form__field">
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="6"
+            maxlength="18"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__group">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="loveCats">
+            Yes
+          </label>
+
+          <label>
+            <input type="radio" name="loveCats">
+            No
+          </label>
+        </div>
+
+        <label class="form__field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form__field">
+          What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+
+        <label class="form__field">
+          What are your favorite brand cars?
+          <select 
+            name="cars"
+            size="4"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+          
+        </label>
+
+        <label class="form__field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__group">
+        <legend>Additional info:</legend>
+
+        <label class="form__field">
+          Comments:
+          <textarea 
+            name="comments"
+            rows="2"
+            cols="20"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to fix this
<img width="305" alt="image" src="https://user-images.githubusercontent.com/53611910/205524653-0487ef35-e934-4f02-b9e9-e4e9add93e03.png">